pkg/eventing/utils.go

```diff
@@ -14,6 +14,10 @@ import (
 
 const publisherProxySuffix = "publisher-proxy"
 
+func GetEPPDeploymentName(eventing v1alpha1.Eventing) string {
```

Comment:
The name EPP is not readable. I think `GetPublisherDeploymentName` is more readable.

Reply:
I told him to use the name you suggested, but @Faizan wanted to keep it because it'd be very long.

Reply:
I have updated the names. https://github.com/kyma-project/eventing-manager/pull/61